### ISSUE TITLE
Don't know how to start libinput-gestures

### ISSUE THREAD

**Issue by Philz69** [bulletmark/libinput-gestures#80]
Everything is working fine if i run libinput-gestures directly.
Libinput-gestures-setup autostart doesnt give any error, but fails to start libinput-gestures. Libinput-gestures-setup start spits out "Dont know how to start libinput-gestures". Libinput-gestures-setup status says autostart is on and libinput-gestures is not started. 

Running latest version from the AUR.

**Comment by bulletmark** [comment]
Please update to latest version and paste the output of `libinput-gestures -e` here.

**Comment by Philz69** [comment]
With the latest version of libinput-gestures from the AUR, it says `libinput-gestures -e` doesnt have a valig argument. 
With the latest version of libinput-gestures-git from the AUR, it outputs
```
libinput-gestures: device /dev/input/event9: Elan Touchpad
libinput-gestures: session i3+x11 on Linux-4.10.11-1-ARCH-x86_64-with-arch, python 3.6.0, libinput 1.7.0
libinput-gestures: using /etc/libinput-gestures.conf with 6 gestures
```

**Comment by Philz69** [comment]
I looked at the code, it is because i dont have gnome nor KDE. I suggest adding this as requirement to the readme, or maybe modifying the failed to start output to say it didnt find any compatible launcher ( KDE / GNOME ). You can close the issue. :)

**Comment by bulletmark** [comment]
Before I posted my previous comment here I released v2.23-1 of libinput-gestures to the AUR. That version has the new '-e` flag. You did not update to the latest AUR version before you tried it.

I realized that you must have been running an environment which does not support the 2 ways I use to start a desktop file so that is why I added the `-e` to report that. Obviously I do not intend this utility to only support GNOME and KDE so the fix is to work out what desktop starter i3 uses and add that to `libinput-gestures-start`.
